cmd/chaos-controller-manager/main.go

```diff
@@ -212,14 +212,7 @@ func Run(params RunParams) error {
 	}
 
 	go watchConfig(configWatcher, conf, controllerRuntimeSignalHandler.Done())
```

Comment:
This line could also be removed.

pkg/webhook/config and pkg/webhook/inject could also be removed safely

Reply:
Done